cmd/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"os"
+	"weather"
+)
+
+func main() {
+
+	weather.RunCLI(os.Args)
```

Comment:
Great! In fact, we don't even really need to pass `os.Args`, do we? The `RunCLI` method is specifically about being a CLI, so we can "allow" it access to things like `os.Args` directly.